tests/blas_test.cpp

```diff
@@ -487,43 +487,43 @@ class BlasTest : public ::testing::TestWithParam<::testing::tuple<int, int>>
         break;
 
       case Kernel::axpy_block:
-        for (int i = 0; i < niter; ++i) blas::axpy(Ar, xmD, ymoD);
+        for (int i = 0; i < niter; ++i) blas::block::axpy(Ar, xmD, ymoD);
```

Comment:
Only because we're here---is there a reason why we didn't have a corresponding host verify call _here_? I'm guessing there's one somewhere in the depths of this file...

Reply:
Sorry, not following this question.  This is in the `benchmark` test, which doesn't do any correctness testing.

Reply:
That answers my question, I looked quickly at the change without verifying if it was in the benchmark or verify test :)